### PR TITLE
Add typography documentation

### PR DESCRIPTION
**Pull request by v-mingho**
Task: task-[421763](https://dev.azure.com/ceapex/Engineering/_boards/board/t/Fox%20Team/Stories/?workitem=421763)

Link: preview-[137](https://design.docs.microsoft.com/pulls/137/atomics/typography.html)

This PR adds documentation for typography. I also added some default padding to examples.

## Testing

1. Visit the preview link and review the page contents.

### CODE REVIEW

File: site/src/atomics/typography.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+<p class="font-style-italic">Italic text</p>
+```
+
+## Font family
+
+The font family can be transformed with the following classes.
+
+```html
+<p class="font-family-monospace">Font Family Monospace</p>
+<p class="font-family-quote">Font Family for block quotes</p>
```
Comment (wibjorn) [inline site/src/atomics/typography.md:59]: I think we can remove this one, this is used for the internals of a very particular component. It's probably doesn't need to be an atomic either, but that is not a critical issue.

File: site/src/scaffold/styles/layout.scss [hunk 2]
```diff
@@ -62,6 +62,12 @@
 	}
 }
 
+.example {
```
Comment (wibjorn) [inline site/src/scaffold/styles/layout.scss:65]: Do we need this? Should we just add some standard padding to `.example`?

File: site/src/atomics/typography.md (hunk 1 above)
Comment (v-mingho) [reply site/src/atomics/typography.md]: Removed

File: site/src/atomics/typography.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+```
+
+## Text decoration
```
[the remark this answers is not in the document]
Comment (v-mingho) [reply site/src/atomics/typography.md]: Updated

File: site/src/scaffold/styles/layout.scss (hunk 2 above)
Comment (v-mingho) [reply site/src/scaffold/styles/layout.scss:65]: Yesterday I was thinking that having some space between each item would be easier on the eyes. I looked at it again today, doesn't look too bad so we can probably live without it. Added some padding to `.example` though.

File: site/src/atomics/typography.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
+---
+title: Typography Atomics
+description: Typography related atomic css classes for the Atlas Design System
+template: standard
+---
+
+# Typography Atomics
+
+The typography scale is designed to for great readability across the platform. This page covers the various classes available to modify text.
```
Comment (wibjorn) [inline site/src/atomics/typography.md:9]: ~~ to~~ for great

File: site/src/atomics/typography.md [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+
+## Font size
+
+There are two avenues for modifying font sizes in Atlas.
+
+#### Heading sizes
+
+The following classes can be used to accurately adopt a heading's size (h1- h6).
```
Comment (wibjorn) [inline site/src/atomics/typography.md:17]: suggested addition: 

These sizes are tied to the default heading sizes render within articles and the [markdown component](../components/markdown.md).

File: site/src/atomics/typography.md (hunk 4 above)
Comment (v-mingho) [reply site/src/atomics/typography.md]: updated

File: site/src/atomics/typography.md [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+template: standard
+---
+
+# Typography Atomics
+
+The typography scale is designed to for great readability across the platform. This page covers the various classes available to modify text.
+
+## Font size
+
+There are two avenues for modifying font sizes in Atlas.
```
[the remark this answers is not in the document]
Comment (v-mingho) [reply site/src/atomics/typography.md]: updated

File: site/src/atomics/typography.md (hunk 5 above)
Comment (v-mingho) [reply site/src/atomics/typography.md:17]: Added

**Review by wibjorn — APPROVED** [review]

**Pull request merged**